src/Connector.SqlServer/Utils/TableDefinitions/EntityCodeExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace CluedIn.Connector.SqlServer.Utils.TableDefinitions;
+
+using CluedIn.Core.Data;
+using CluedIn.Core.Reflection;
+
+using System.Reflection;
+
+internal static class EntityCodeExtensions
+{
+    public static object IsOriginEntityCode(this IEntityCode code)
```

Comment:
nit: Name looks a bit weird, as it return either `bool` or `DBNull.Value`. Should it be normalized to DbValue or something and later reflected somehow in the name?